app/helpers/storage_service_helper/textual_summary.rb

```diff
@@ -15,7 +15,7 @@ def textual_group_properties
 
   def textual_group_capabilities
     capabilities = @record.capabilities.map do |c|
-      [c["name"], c["value"]]
+      [c.first, c.last]
     end
```

Comment:
What does `@record.capabilities` look like beforehard? Is c a simple array of 2 items? or is capabilities a simple key -> value hash?

If a simple array of 2 times, I don't think you need to map at all.  If it's a hash this can just be `.to_a`

Reply:
Hi, that's taken from #8656 
Otherwise the page won't be rendered to begin with, since you've already merged https://github.com/ManageIQ/manageiq-providers-autosde/pull/213.
If this line needs change I think we better change it there and rebase this PR on master once that PR is merged. @galoiring 

But to answer your question, service capabilities look like this:
```
{
:compression  => "true",
:thin_provision => "true"
}
```
So will `.to_a` give us this?
```
[
["compression", "true"], 
["thin_provision", "true"]
]
```